var/spack/repos/builtin/packages/draco/package.py

```diff
@@ -64,13 +68,14 @@ class Draco(CMakePackage):
     depends_on('superlu-dist@:5.99', when='@:7.6.99+superlu_dist')
 
     conflicts('+cuda', when='@:7.6.99')
+    conflicts('+caliper', when='@:7.7.99')
 
     # Fix python discovery.
     patch('d710.patch', when='@7.1.0^python@3:')
     patch('d710-python2.patch', when='@7.1.0^python@2.7:2.99')
     patch('d730.patch', when='@7.3.0:7.3.99')
     patch('d740.patch', when='@7.4.0:7.4.99')
-    patch('smpi.patch', when='@:7.6.99')
+    patch('d760-cray.patch', when='@7.6.0')
```

Comment:
Is `smpi.patch` used at all now? should we delete it from the repository?

Reply:
> Is `smpi.patch` used at all now? should we delete it from the repository?

@alalazo That looks like a bug on my part.  Let me recheck the whole file to make sure I didn't mess something else up.  Thanks for noticing!

Reply:
@alalzo Indeed, older versions of draco are no longer supported for `^spectrum-mpi` and newer versions work without the patch.  I've deleted the excess file in my updated PR.